apps/checkout/src/sections/Addresses/Addresses.tsx

```diff
@@ -22,18 +23,30 @@ export const Addresses: React.FC = () => {
   return (
     <div>
       {checkout.isShippingRequired && (
-        <ShippingAddressSection
+        <CountrySelectProvider
+          selectedCountryCode={
+            checkout?.shippingAddress?.country?.code as CountryCode
+          }
+        >
+          <ShippingAddressSection
+            addresses={userAddresses}
+            defaultShippingAddress={user?.defaultShippingAddress}
+            useShippingAsBillingAddress={useShippingAsBillingAddress}
```

Comment:
Might be worth renaming it so it doesn't sound like a hook. 'isBillingSameAsShippingAddress'

Reply:
Done